milestones/tests/test_services.py

```diff
@@ -0,0 +1,39 @@
+"""
+Test for the xBlock service
+"""
+
+import unittest
+import types
+
+from milestones.services import (
+    MilestonesService
+)
+from milestones import api as milestones_api
+
+
+class TestMilestonesService(unittest.TestCase):  # pylint: disable=too-many-public-methods
```

Comment:
Is it complaining about the number of public methods on the `MilestonesService` class or the number of methods on the `TestMilestonesService` class?  I assume the latter.  
We aren't creating a new test method for each function - we are just asserting for each function.  So I'm still missing what it's complaining about.